qiskit/optimization/algorithms/admm_optimizer.py

```diff
@@ -98,7 +114,26 @@ def __init__(self,
         self.continuous_indices = continuous_indices
         self.sense = op.objective.get_sense()
 
+        # define heavily used matrix, they are used at each iteration, so let's cache them,
+        # they are np.ndarrays
+        # objective
```

Comment:
Allow these cache names
```suggestion
        # pylint:disable=invalid-name 
        # objective
```

Reply:
Regarding the problem conversion, let's first fix everything else and merge it and then in a second step look at that.